rules/windows/discovery_admin_recon.toml

```
@@ -14,7 +14,43 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Enumeration of Administrator Accounts"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating Enumeration of Administrator Accounts
+
+After a successful initial compromise, attackers may try to gain situational awareness of the environment to plan their
+next steps. This can happen by running commands to enumerate network resources, users, connections, files, and security
+software installed.
+
+This rule looks for the execution of the `net` and `wmic` utilities to enumerate administrator-related users or groups 
+in both domain and local scope, this information can be used by attackers to decide the next steps of the attack, such
+as mapping targets for credential compromise and other post-exploitation activities.
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree).
+- Identify the user account that performed the action and whether it usually performs this kind of action.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Investigate abnormal behaviors done using the account, such as commands executed, startup folder modifications, and
```

```suggestion
- Investigate abnormal behaviors observed using the account, such as commands executed, startup folder modifications, and
```

May want to make this suggestion throughout the other files?